Localize/localize-pipeline.yml

```diff
@@ -28,8 +28,16 @@ stages:
       
     - powershell: |
         $sprintInfo = Invoke-WebRequest https://whatsprintis.it -Headers @{"Accept"= "application/json"} | ConvertFrom-Json
-        Write-Host "##vso[task.setvariable variable=week]$($sprintInfo.week)"
-        Write-Host "##vso[task.setvariable variable=sprint]$($sprintInfo.sprint)"
```

Comment:
'sprint' variable is required for task bump further

Reply:
Oh, yeah you are right missed that here.